dll/win32/shell32/shelldesktop/CDirectoryWatcher.cpp

```diff
@@ -171,37 +170,19 @@ void CDirectoryWatcher::ProcessNotification()
         // convert action to event
         fDir = PathIsDirectoryW(szPath);
         dwEvent = ConvertActionToEvent(pInfo->Action, fDir);
-
-        // convert SHCNE_DELETE to SHCNE_RMDIR if the path is a directory
-        if (!fDir && (dwEvent == SHCNE_DELETE) && m_dir_list.ContainsPath(szPath))
-        {
-            fDir = TRUE;
-            dwEvent = SHCNE_RMDIR;
```

Comment:
Is this event still created?
The shell expects this..

Reply:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&sr=1&s=SHCNE_RMDIR

There are multiple users of `SHCNE_RMDIR`